### DISCUSSION TITLE
Ironwail 0.4.0

### DISCUSSION THREAD

**Discussion by andrei-drexler** [andrei-drexler/ironwail#31] [Announcements]
### Changes from previous release:
- added support for multiple game and basegame dirs (from QSS, contributed by @neogeographica)
- added support for Unicode characters in basedir
- changed config file name to `ironwail.cfg` (`config.cfg` will be loaded as a fallback when `ironwail.cfg` is not present)
- changed screenshot prefix from `spasm` to `ironwail`
- changed r_softemu 3 menu background to look more like the one in Quake 1.06
- changed r_softemu 3 mode to replicate the alias model UV distortion from the original software renderer
- restored r_showbboxes functionality
- added map status to window title
- added fps limit menu option
### Bug fixes:
- fixed a bug that caused an edict leak after loading a savegame in some cases
- fixed a bug that caused physics to run in sync with the renderer when using the default host_maxfps limit
- fixed a bug that caused water to be drawn as opaque instead of transparent in some cases

<hr /><em>This discussion was created from the release <a href='https://github.com/andrei-drexler/ironwail/releases/tag/v0.4.0'>Ironwail 0.4.0</a>.</em>

**Comment by garoto** [comment]
Was meaning to ask for a while, but why does your fork doesn't ship a `quakespasm.pak` by default? Is it because you have `gl_zfix` set to 1 by default?

**Reply by andrei-drexler** [reply]
That's basically it, I'm not including the pak because it's no longer needed and I want to keep the release files as simple as possible. `gl_zfix 1` (custom implementation) makes the ent patches redundant. `default.cfg` is already included in the exe. That only leaves the console background - it looks good, but I can live without it.

**Reply by garoto** [reply]
Got it, thanks.
